flexmeasures/ui/templates/crud/account.html

```diff
@@ -0,0 +1,35 @@
+{% extends "base.html" %}{% block title %} {% endblock %} {% block divs %} {% if account %}
+
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-sm-8">
+            <div class="account-data-table card">
+                <h2>Account overview</h2>
+                <small>Account: {{ account["name"] }}</small>
+                <table class="table table-striped table-responsive">
+                    <tbody>
+                        <tr>
+                            <td>
+                                ID
+                            </td>
+                            <td>
+                                {{ account["account_id"] }}
```

Comment:
"account_id" or just "id"?

I'd also prefer `account.id`. Dot notation might work here regardless of whether Jinja is passed an object or dictionary.